src/openMVG/sfm/sfm_data_triangulation.cpp

```diff
@@ -335,13 +365,13 @@ const
         continue;
       const IntrinsicBase * cam = sfm_data.GetIntrinsics().at(view->id_intrinsic).get();
       const Pose3 pose = sfm_data.GetPoseOrDie(view);
-      bChierality &= CheiralityTest((*cam)(itObs->second.x), pose, X);;
+      bCheirality &= CheiralityTest((*cam)(itObs->second.x), pose, X);
       const Vec2 residual = cam->residual(pose(X), itObs->second.x);
       bReprojection_error &= residual.squaredNorm() < dSquared_pixel_threshold;
-      validity_test_count += (bChierality && bReprojection_error) ? 1 : 0;
+      validity_test_count += (bCheirality && bReprojection_error) ? 1 : 0;
     }
 
-    if (!bChierality || !bReprojection_error ||
```

Comment:
I have made the typo `chierality` instead of `cheirality` in 24 places across the project (my bad)

Reply:
I never payed attention before, "Chierality" sounds good in french.

Reply:
:fr: ;-)